mitodl/mitxpro
# Added API calls to validate user for export compliance

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #62 

#### What's this PR do?
Adds the API call to make a check to SDN and updated the auth API/ui to support this.

#### How should this be manually tested?
- Configure the cybersource env vars added in this PR
- Register a new user, provide a valid user/name, confirm you can still complete registration
- Register with user details that will trigger a failure:
  - firstName: `Mohammed`
  - lastName: `QASIM`
  - street: `1295 Charleston Road`
  - city: `Mountain View`
  - state: `CA`
  - postalCode: `94043`
  - country: `US`

#### Screenshots (if appropriate)

![Screenshot_2019-05-10 mitxpro](https://user-images.githubusercontent.com/28598/57551444-125d2300-7337-11e9-9934-93cf91f0f798.png)
